recipe/meta.yaml

```diff
@@ -21,26 +21,36 @@ build:
   script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir --no-build-isolation -vvv
 
 requirements:
+  build:
+    # we use go directly because we do not have activation feedstocks
+    # therefore we can't use compiler('go') and we skip linting
```

Comment:
Won't it work with `compiler('go')`? Asking, because our cbc.yaml provides the version we support https://github.com/AnacondaRecipes/aggregate/blob/master/conda_build_config.yaml#L102-L117